spring-cloud-gcp-data-spanner/src/main/java/com/google/cloud/spring/data/spanner/core/SpannerMutationFactoryImpl.java

```diff
@@ -177,7 +177,7 @@ private void verifyChildHasParentId(SpannerPersistentEntity parentEntity,
 	}
 
 	private WriteBuilder writeBuilder(Op op, String tableName) {
-		Mutation.WriteBuilder builder = null;
+		Mutation.WriteBuilder builder;
```

Comment:
Can't wait for switch statements returning values. Is spacing correct on the case statements? They look oddly aligned.

Reply:
Yep - they're just tabs.